store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/FederatedStore.java

```diff
@@ -16,6 +16,8 @@
 
 package uk.gov.gchq.gaffer.federatedstore;
 
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import org.apache.htrace.fasterxml.jackson.annotation.JsonCreator;
```

Comment:
This is the wrong import. This is a shaded version from a different JAR and we cannot control the version.
```suggestion
import com.fasterxml.jackson.annotation.JsonCreator;
```